package.json

```diff
@@ -82,6 +82,7 @@
     "coveralls": "nyc report --reporter=text-lcov | coveralls",
     "dev": "rimraf ./lib && babel -w src --out-dir lib --copy-files",
     "lint": "eslint src/**",
+    "lint-test": "eslint test/",
```

Comment:
shouldn't this be also included in travis file to be executed in CI?

Reply:
The line above can just be change from 
```
"lint": "eslint src/**",
```
to
```
"lint": "eslint '{src,test}/**/*.js'",
```

Reply:
Fixed.